### DISCUSSION TITLE
Gamification : Managing Domains and Rules

### DISCUSSION THREAD

**Discussion by sergeByishimo** [exoplatform/exo-community#28] [Q&A]
eXo Platform Community 6.4.1
Gamification :: Portlets Pack Version: 2.4.1-exo

I can not see how to manage domains and rules.

In this latest [documentation](https://exo-documentation.readthedocs.io/en/latest/Administration.html?highlight=domains#manage-domains) they show options to manage domains and rules.

![Screen Shot 2023-08-07 at 12 42 26](https://github.com/Meeds-io/meeds/assets/32865999/1beb3162-793c-42d5-a201-1877e925e9f9)
 
BUT I can not see the options as you can see, I can only manage Badges.

![Screen Shot 2023-08-07 at 12 47 33](https://github.com/Meeds-io/meeds/assets/32865999/3fcc1965-e479-459f-ae26-3b7954e597aa)

**Comment by ahamdi** [comment]
Hi @sergeByishimo 
This documentation is no more accurate and will be replaced by the new documentation in https://docs.exoplatform.org
The management of the Gamification functionality was moved to the contribution center, you can access it from the Applications menu ion the left at the URL http://YOUR_DOMAIN/portal/dw/contributions

**Reply by jonasbrant** [reply]
is there a video explaining how to configurate or its potential?

**Reply by rdenarie** [reply]
@srenault-meeds Have you some video links for @jonasbrant ?

Thanks

**Reply by srenault-meeds** [reply]
Sure. It depends on which version you use. @jonasbrant can you please confirm me the version of your platform?
Thanks

**Reply by jonasbrant** [reply]
6.4 @srenault-meeds

**Comment by srenault-meeds** [comment]
Ok. I see. The Contributions Center has been completely reviewed for the next version of eXo. It is planned to be early 2024. 
I can try to provide you a video though, so you can start creating those.

The new version will provide a better way to organize incentive actions and related programs. And finally, programs and related actions will be more visible to users with widgets and dedicated page as you can see on builders for example.

What is your timeline regarding this?
What kind of programs you'd like to propose to your users?

Thanks

**Reply by jonasbrant** [reply]
Samuel, the team just let me know here that we are 6.5. 
But we can wait until early 2024;

**Comment by srenault-meeds** [comment]
Not possible I think because the 6.5 GA is not available yet
